client/src/main/java/hudson/plugins/swarm/Options.java

```diff
@@ -209,4 +209,9 @@
             name = "-prometheusPort",
             usage = "If defined, then start an HTTP service on this port for Prometheus metrics.")
     public int prometheusPort = -1;
+
+    @Option(
+            name = "-config",
+            usage = "Yaml configuration file containing the options.")
```

Comment:
I think YAML should be capitalized here.

Reply:
Fixed.